docs/dev.rst

```diff
@@ -0,0 +1,490 @@
+            - github: https://github.com/SalesforceFoundation/EDA
+              unmanaged: True
+
+The EDA repository is configured for a namespace, but the dependency  specifies ``unmanaged: True``, so EDA and its dependencies deploy as unmanaged metadata.
+
+
+
+Reference a Specific Tag
+************************
+
+To reference a specific version of the product other than the most recent commit on the main branch (for unmanaged projects) or the most recent production release (for managed packages), use the ``tag`` option to specify a tag from the target repository. This option is most useful for testing against beta versions of underlying packages, or recreating specific org environments for debugging.
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - github: https://github.com/SalesforceFoundation/EDA
+              tag: beta/1.47-Beta_2
+
+The EDA repository's tag, ``beta/1.47-Beta_2``, is used instead of the latest production release of EDA (1.46, for this example). This tag lets a build environment use features in the next production release of EDA that are already merged but not yet included in a production release.
+
+
+
+Skip ``unpackaged/*`` in Reference Repositories
+***********************************************
+
+If the referenced repository has dependency metadata under ``unpackaged/pre`` or ``unpackaged/post``, use the ``skip`` option to skip deploying that metadata with the dependency.
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - github: https://github.com/SalesforceFoundation/EDA
+              skip: unpackaged/post/course_connection_record_types
+
+
+
+Managed Package Dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Managed package dependencies are rather simple. Under the ``project__dependencies`` section of the ``cumulusci.yml`` file, specify the namespace of the target package, and the required version number.
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - namespace: npe01
+              version: 3.6
+
+
+
+Automatic Install, Upgrade, or Uninstall/Install
+************************************************
+
+When the ``update_dependencies`` task runs, it retrieves a list of all managed packages in the target org, and creates a list of the installed packages and their version numbers.
+
+.. code-block:: yaml
+
+    project:
+        dependencies:
+            - namespace: npe01
+              version: 3.6
+    
+    
+Depending on whether or not the package with namespace ``npe01`` is installed, the ``update_dependencies`` task runs these steps. 
+
+* If ``npe01`` is not installed, ``npe01 version 3.6`` is installed.
+* If the org already has ``npe01 version 3.6`` installed, no changes take place.
+* If the org has an older version installed, it's upgraded to ``version 3.6``.
+* If the org has a newer version or a beta version installed, it's uninstalled and ``version 3.6`` is installed.
+
+
+
+Hierarchical Dependencies
```

Comment:
I'd like to remove this section also as this functionality is going to be deprecated and thereafter removed.

Reply:
done.